sw/device/silicon_owner/tock/tests/basic/BUILD

```diff
@@ -39,6 +39,10 @@ opentitan_test(
     name = "basic_test",
     cw310 = cw310_params(
         binaries = {":image": "firmware"},
+        tags = [
+            "broken",
+            "manual",
+        ],
```

Comment:
The Tock test is disabled for now, but most likely, this isn't the best repo for it. I believe RTL changes in this repo should not be gated by a simultaneous Tock support requirement. In terms of scheduling and resources, Tock support is downstream (and probably should depend on this repo's _releases_, rather than keeping pace with each PR).

Reply:
Does this deserve a comment inline so that it is clear why this has been marked disabled/broken?

Reply:
I'm not sure. I think we should remove this test from the repo altogether, really.

It's broken because a third-party software repo doesn't support earlgrey, but it seems a little funky to have this repo carry issues for that one.

Reply:
I'm ok with removing it, since it creates a weird dependency loop.